src/woo_publications/metadata/api/filterset/category.py

```diff
@@ -0,0 +1,28 @@
+from django.utils.translation import gettext_lazy as _
+
+from django_filters.rest_framework import FilterSet, filters
+
+from woo_publications.api.custom_filters import URLFilter
+from woo_publications.metadata.models import InformationCategory
+
+
+class InformationCategoryFilterset(FilterSet):
+    identifier = URLFilter(
+        lookup_expr="exact",
+        help_text=_(
+            "Search the information category based on the unique URI that identifies a specific category."
+        ),
+    )
+    naam = filters.CharFilter(
+        lookup_expr="exact",
+        help_text=_(
+            "Search the information category based on the name of the category."
+        ),
+    )
```

Comment:
for auto-complete purposes this will need to support partial matches, so the `lookup_expr` should be `icontains`. Please add a test for this too.